functional/FunctionalTest.java

```diff
@@ -3158,14 +3168,11 @@ public static void composeObject_test1() throws Exception {
         client.removeObject(
             RemoveObjectArgs.builder().bucket(bucketName).object(destinationObjectName).build());
       } finally {
-        Files.delete(Paths.get(filename1));
-        Files.delete(Paths.get(filename2));
-        client.removeObject(
-            RemoveObjectArgs.builder().bucket(bucketName).object(filename1).build());
-        client.removeObject(
-            RemoveObjectArgs.builder().bucket(bucketName).object(filename2).build());
+        Map<String, List<String>> deleteObjectList = new HashMap<String, List<String>>();
+        deleteObjectList.put(
+            bucketName, new ArrayList<String>(Arrays.asList(filename1, filename2)));
+        deleteFilesAndObjects(deleteObjectList);
```

Comment:
If you go with my previous suggestion, this block will become far simpler:

`deleteFilesAndObjects(bucketName, new String[]{filename1, filename2});`